invenio.cfg

```diff
@@ -841,20 +841,55 @@ PREVIEWER_PREFERENCE = [
 """Preferred previewers."""
 
 MIRADOR_PREVIEW_EXTENSIONS = [
-    "gif",
     "png",
     "jp2",
     "jpeg",
     "jpg",
     "png",
     "tif",
+    "tiff",
 ]
 """Mirador previewer allowed extensions."""
 
 MIRADOR_PREVIEW_CONFIG = {
+
+  "window": {
+    "allowClose": False,
+    "allowFullscreen": True,
+    "allowTopMenuButton": True,
+    "allowMaximize": False,
+    "allowWindowSideBar": True,
+    "defaultSidebarPanelHeight": 201,
+    "defaultSidebarPanelWidth": 235,
+    "defaultView": "xsingle",
+    "hideWindowTitle": True,
+    "sideBarOpen": False,
+    "switchCanvasOnSearch": False,
+    "panels": {
+      "info": True,
+      "attribution": False,
+      "canvas": False,
+      "annotations": False,
+      "search": False,
+      "layers": False
+    },
+    "views": [
+      { "key": "single", "behaviors": ["individuals"] },
+      { "key": "book", "behaviors": ["paged"] },
+      { "key": "scroll", "behaviors": ["continuous"] },
+      { "key": "gallery" }
+    ]
+    },
     "workspace": {
         "draggingEnabled": True,
         "showZoomControls": True,
     }
 }
 """Config for the Mirador previewer."""
+
+IIIF_TILES_GENERATION_ENABLED = True
+
+IIIF_TILES_VALID_EXTENSIONS = [ "pdf","png", "jp2", "jpeg", "jpg", "png", "tif", "tiff"]
```

Comment:
comment: I think these get eventually normalized, but maybe it's worth being explicit about all extension types.

Reply:
would `IIIF_TILES_VALID_EXTENSIONS =["pdf","png", "jpeg", "jpg", "tiff"]` suffice?

Reply:
I somewhat like having them "expanded". E.g. I'm not sure if we include `jp2` in the original config, and now it's too late to fix :)